duration visibility_time
duration invisibility_time
uint32 visibility_frame_count
uint32 invisibility_frame_count
duration moving_time
duration standing_time
duration moving_time_sum
duration standing_time_sum
float64 min_linear_speed
float64 max_linear_speed
